packages/core/src/database/sqlite/service.ts

```diff
@@ -51,6 +51,8 @@ export class SqliteDatabaseService implements BaseDatabaseService {
   private userNamespace: string;
   private internalNamespace: string;
 
+  userDatabaseFile: string;
```

Comment:
I think for SQLite we should open a new readonly database connection.

https://github.com/WiseLibs/better-sqlite3/blob/master/docs/api.md#new-databasepath-options